### DISCUSSION TITLE
rtl_airband is not working with libSoapySDR.so.0.8

### DISCUSSION THREAD

**Discussion by ageras1** [rtl-airband/RTLSDR-Airband#266] [Q&A]
first time here i don't know if it is the right place to post this. i am using arch linux and the rtlsdr-airband-git aur pkg (it compiles the latest version if i am not mistaken) is not working with libSoapySDR.so.0.8. i compiled libSoapySDR.so.0.7 manually but then soapysdr did not work with the latest soapysdrplay and the earlier version of soapysdrplay does not exist anymore (i can't find it). i thought to bring this "issue" here if you are not aware about it but mostly if i am missing out something or if there is a work around. thanks for your time.

**Comment by szpajder** [comment]
rtl_airband supports SoapySDR 0.8 since version 3.2.0.

rtlsdr-airband-git from AUR (https://aur.archlinux.org/packages/rtlsdr-airband-git) builds the latest unstable branch, which also supports SoapySDR 0.8. However the build procedure in unstable branch has changed a few months ago, but the AUR pkg has not been updated to reflect that (it is flagged as out of date, BTW).

Just ditch the AUR pkg, clone the sources from git (or fetch the latest stable release) and build the program manually.

**Reply by ageras1** [reply]
I try to test things out  before asking for help or something but my understanding of most things goes that far.
It didn't pass my mind to just build the program manually, the convenience of the aur pkg as great as it is ... i was not aware that this could happen. i just learned something, i am sorry if it was mentioned somewhere and i just skipped. Thank you for you time.

**Comment by israelgoldstein** [comment]
im facing an issue where i cant use airspy mini in version 4.0.3, 
do you have some fix for it? or its a separate issue ?

**Reply by charlie-foxtrot** [reply]
open a new issue please

**Reply by israelgoldstein** [reply]
done
https://github.com/charlie-foxtrot/RTLSDR-Airband/issues/385
